packages/nx/src/utils/package-manager.ts

```diff
@@ -126,7 +133,6 @@ export function getPackageManagerCommand(
       };
     },
     npm: () => {
-      // TODO: Remove this
```

Comment:
Please revert this, I did not ask you to remove the existing comment. I asked you to remove the copy of this comment you added fresh in your PR